libs/openFrameworks/events/ofEvent.h

```diff
@@ -47,6 +47,14 @@ class ofBaseEvent{
     	enabled = false;
     }
 
+	bool isEnabled() const {
+		return enabled;
+	}
+
+	std::size_t size() const {
+		return functions.size();
```

Comment:
@arturoc I've been thinking about this and wondering -- since we're accessing the functions array -- should this actually be:

```c++
std::size_t size() const {
    std::unique_lock<std::mutex> lck(mtx);
    return functions.size();
}

```

We'd also have to make `mtx` mutable.